sdk/core/core-http/package.json

```diff
@@ -133,7 +133,7 @@
     "@azure/core-tracing": "1.0.0-preview.8",
     "@azure/logger": "^1.0.0",
     "@opentelemetry/api": "^0.6.1",
-    "cross-env": "^6.0.3",
+    "cross-env": "^7.0.2",
```

Comment:
@xirzec, @daviwil: I noticed that `cross-env` is a runtime dep of `core-http`, while it's a dev dep of other packages.  Bug or by design?

Reply:
Pretty sure this is a bug, introduced here:

https://github.com/Azure/azure-sdk-for-js/pull/7500/files#diff-b2f882ebd30b5e12e5e4ff490489c7d8R128

I will fix this in a separate PR.

CC: @willmtemple